torch/onnx/_internal/_exporter_legacy.py

```diff
@@ -35,6 +34,7 @@
     patcher as patcher,
     registration,
 )
+from torch.utils.pytree import tree_any
```

Comment:
```suggestion
from torch.utils import pytree
```

Is it possible to import the pytree module? We would like to import modules only under onnx/